pkg/micro-ecc/psa_uecc/p192.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2021 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_psa_crypto pkg_micro-ecc
+ * @{
+ *
+ * @brief       Glue code translating between PSA Crypto and the Micro-ECC APIs
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include "psa/crypto.h"
+#include "uECC.h"
+
+psa_status_t psa_generate_ecc_p192r1_key_pair(  const psa_key_attributes_t *attributes,
```

Comment:
Is there a given guarantee from the upper layer that all these pointers will always be valid? Otherwise I'd keep my defensive approach 🛡️ and check them

Reply:
The attributes are checked first thing in `psa_crypto.c` and all key buffers come from inside the implementation, not from the user or application. So they'll be valid here =)